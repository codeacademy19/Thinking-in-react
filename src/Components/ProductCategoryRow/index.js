import React, { Component } from 'react';
class ProductCategoryRow extends Component {
    render() {
      //const category = this.props.category;
      return (
        <tr>
          <th colSpan="2">
            {this.props.category}
          </th>
        </tr>
      );
    }
  }
  export default ProductCategoryRow;