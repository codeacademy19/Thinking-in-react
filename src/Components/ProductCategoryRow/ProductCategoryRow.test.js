import React from 'react';
import renderer from 'react-test-renderer';
import ProductCategoryRow from '.';
import {shallow} from 'enzyme';


describe('ProductCategoryRow',()=>{
    it('should render a row with value as Product Category',()=>{
        const tree=renderer.create(<ProductCategoryRow category="Sporting Goods" />).toJSON();
        expect(tree).toMatchSnapshot();
    })
});