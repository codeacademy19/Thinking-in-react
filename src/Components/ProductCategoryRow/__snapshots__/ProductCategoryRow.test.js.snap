// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductCategoryRow should render a row with value as Product Category 1`] = `
<tr>
  <th
    colSpan="2"
  >
    Sporting Goods
  </th>
</tr>
`;
